File: updated_code/code/java/119-pascals-triangle-ii.java
import java.util.ArrayList;
import java.util.List;

/**
 * This class calculates the `rowIndexth` (0-indexed) row of Pascal's triangle.
 */
public class Solution {
    /**
     * Calculate the `rowIndexth` (0-indexed) row of Pascal's triangle.
     * 
     * @param rowIndex - The index of the row in Pascal's triangle.
     * @return The `rowIndexth` row of Pascal's triangle.
     */
    public List<Integer> getRow(int rowIndex) {
        List<Integer> row = new ArrayList<>();
        row.add(1);
        if (rowIndex == 0) {
            return row;
        }
        row.add(1);
        for (int i = 2; i <= rowIndex; i++) {
            List<Integer> newRow = new ArrayList<>();
            newRow.add(1);
            for (int j = 1; j < i; j++) {
                newRow.add(row.get(j - 1) + row.get(j));
            }
            newRow.add(1);
            row = newRow;
        }
        return row;
    }
}

File: updated_code/code/python/119-pascals-triangle-ii.py
Python
from typing import List

class Solution:
    """
    Class with the solution.
    
    """

    def get_row(self, row_index: int) -> List[int]:
        """
        Returns the rowIndexth row of Pascal's triangle.
        
        Each number in the triangle is equal to the sum of the two numbers directly above it.
        
        :param row_index: The index of the row to be returned.
        :return: The rowIndexth row of Pascal's triangle.
        
        """
        row = [1]
        for _ in range(row_index):
            row = [x + y for x, y in zip([0]+row, row + [0])]
        return row